addons/arsenal/stringtable.xml

```diff
@@ -155,115 +155,138 @@
             <French>Liste d'équipements de base exportée dans le presse papier</French>
+            <German>Standard-Ausrüstungen-Liste in die Zwischenablage exportiert</German>
             <Polish>Lista domyślnych zestawów została eksportowana do schowka</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_exportCurrent">
             <English>Current loadout exported to clipboard</English>
             <French>Équipement actuel exporté dans le presse papier</French>
+            <German>Derzeitige Ausrüstung in die Zwischenablage exportiert</German>
             <Polish>Obecny zestaw został eksportowany do schowka</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_importFormatError">
             <English>Wrong format provided</English>
             <French>Mauvais format fourni</French>
+            <German>Falsches Format verwendet</German>
             <Polish>Podano zły format</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_importedDefault">
             <English>Default loadouts list imported from clipboard</English>
             <French>Liste d'équipements de base importée depuis le presse papier</French>
+            <German>Standard-Ausrüstungen-Liste aus der Zwischenablage importiert</German>
             <Polish>Lista domyślnych zestawów została importowana ze schowka</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_importedCurrent">
             <English>Loadout imported from clipboard</English>
             <French>Équipement importé depuis le presse papier</French>
+            <German>Ausrüstung aus der Zwischenablage importiert</German>
             <Polish>Zestaw został importowany ze schowka</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_loadoutDeleted">
             <English>The following loadout was deleted:</English>
             <French>L'équipement suivant fut supprimé:</French>
+            <German>Folgende Ausrüstung wurde entfernt:</German>
             <Polish>Następujący zestaw został skasowany:</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_loadoutUnshared">
             <English>The following loadout is not public anymore:</English>
             <French>L'équipement suivant n'est plus public:</French>
+            <German>Folgende Ausrüstung ist nicht mehr öffentlich:</German>
             <Polish>Następujący zestaw nie jest już publiczny:</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_saveEmptyNameBox">
             <English>The name field is empty!</English>
             <French>Le champ nom est vide !</French>
+            <German>Das Feld "Name" ist leer!</German>
             <Polish>Pole nazwy jest puste!</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_saveAuthorError">
             <English>You are the author of this loadout</English>
             <French>Vous êtes l'auteur de cet équipement</French>
+            <German>Du bist der Ersteller dieser Ausrüstung</German>
```

Comment:
Autor would also fit here. All other translations use Autor instead of creator

Reply:
Autor klingt eher nach einem Schrift- oder Musikstück. Deswegen hatte ich mich gegen Autor entscheiden. Im Englischen funktioniert author für diesen Anwendungsfall.